hack/verify-diff.sh

```diff
@@ -1,10 +1,22 @@
 #!/bin/bash
-FILE_DIFF=$(git ls-files -o --exclude-standard)
 
-if [ "$FILE_DIFF" != "" ]; then
-  echo "Found untracked files:"
-  echo "$FILE_DIFF"
-  exit 1
+status=0
+# Checks that there are no uncommitted changes.
+if [[ -n "$(git status --untracked-files=no --porcelain .)" ]]; then
```

Comment:
please remove  `-untracked-files=no` and find out why you don't see the added files locally (I see them after `make go-tidy go-vendor`)

/hold

Reply:
also, please put such workarounds in a commit on its own for better visibilty